__tests__/app.test.js

```diff
@@ -329,4 +329,53 @@ describe("GET /api/users", () => {
             })
         })
     })
+})
+
+describe("GET /api/articles?topic", () => {
+    test("status 200: responds with articles of specified topic and if topic query is omitted, responds with all articles", () => {
```

Comment:
Please update the 400 test to a 404 and add another 200 test to check that we get back an empty array when an existing topic has no associated articles.